src/Framework/FieldsAPI/WpEditor.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Give\Framework\FieldsAPI;
+
+/**
+ * @unreleased
+ */
+class WpEditor extends Field {
```

Comment:
Please rename this to `WPEditor` — abbreviations should always be capitalized, even in studly-casing.

Reply:
class renamed: https://github.com/impress-org/givewp/compare/d248078a3ac9...ddc803ea03ad